x-pack/plugins/maps/public/classes/styles/vector/style_meta.ts

```diff
@@ -20,7 +20,9 @@ export class StyleMeta {
   }
 
   getCategoryFieldMetaDescriptor(fieldName: string): Category[] {
-    return this._descriptor.fieldMeta[fieldName].categories;
+    return this._descriptor.fieldMeta[fieldName] && this._descriptor.fieldMeta[fieldName].categories
```

Comment:
You only need check for this._descriptor.fieldMeta[fieldName]. [FieldMeta](FieldMeta) defines `categories` as always provided so there is no reason to check for `categories`.
```
return this._descriptor.fieldMeta[fieldName]
      ? this._descriptor.fieldMeta[fieldName].categories
      : [];
``